ddtrace/context.py

```diff
@@ -76,11 +76,9 @@ def clone(self):
         """
         with self._lock:
             new_ctx = Context(
-                trace_id=self._parent_trace_id,
-                span_id=self._parent_span_id,
                 sampling_priority=self._sampling_priority,
             )
-            new_ctx._current_span = self._current_span
+            new_ctx._set_current_span(self._current_span)
```

Comment:
@Kyle-Verhoog this is to consolidate the handling of propagation of span properties

Reply:
I like it, but I think we should pull it into a separate PR 🙂

Reply:
https://github.com/DataDog/dd-trace-py/pull/400 I think just need to re-run tests